<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QDialog" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdditem">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewinv">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View Inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonlogout">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>250</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetInv">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxItemType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Product</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Supply</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Brand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrand">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCategory">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditsup">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxConsumable">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Consumable?</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelItemID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItemID">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDesc">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxQuantity">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quatity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPrice">
   <property name="geometry">
    <rect>
     <x>741</x>
     <y>75</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReport">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
